.github/workflows/ios_build_hotfix_release.yml

```diff
@@ -48,6 +49,16 @@ jobs:
             *) echo "👎 Not a hotfix release branch"; exit 1 ;;
           esac
 
+      - name: Setup Ruby and dependencies
+        uses: ./.github/actions/setup-ruby
+        with:
+          working-directory: iOS
+          cache-key-prefix: ios-ruby
+
+      - name: Extract Asana Task ID
+        id: task-id
+        run: bundle exec fastlane run asana_extract_task_id task_url:"${{ github.event.inputs.asana-task-url }}"
```

Comment:
Moving extracting task URL to the first job in the workflow in order to be able to report failure of every subsequent job in this workflow. The same is done for macOS in the hotfix workflow.